includes/polling/applications/memcached.inc.php

```diff
@@ -1,6 +1,53 @@
 <?php
 
-$data = $agent_data['app']['memcached'][$app['app_instance']];
+if (!empty($agent_data['app']['memcached'])) {
+    $data = $agent_data['app']['memcached'][$app['app_instance']];
+} else {
+    $options = '-O qv';
+    $oid     = 'nsExtendOutputFull.9.109.101.109.99.97.99.104.101.100';
+    $res     = explode(';' , snmp_get($device, $oid, $options));
```

Comment:
Wouldn't it be easier to remove the first line `<<<app-memcached>>>`, and then call `unserialize()` on the remainder of the string?